User/Makefile

```diff
@@ -101,8 +101,8 @@ ifeq ($(FUZZ_MEM),)
 endif
 
 ifeq ($(COVERAGE),1)
-	CFLAGS  += -fprofile-arcs -ftest-coverage -D COVERAGE_TEST
```

Comment:
What is the difference between these two flags and the new one? Is it supported by gcc?

Reply:
--coverage is just a shorter synonym for those two options and yes, it is supported by gcc.
https://gcc.gnu.org/onlinedocs/gcc/Instrumentation-Options.html